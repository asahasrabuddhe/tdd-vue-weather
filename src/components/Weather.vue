<template>
    <div>
        Weather
    </div>
</template>

<script>
    export default {
        name: "Weather"
    }
</script>

<style scoped>

</style>
