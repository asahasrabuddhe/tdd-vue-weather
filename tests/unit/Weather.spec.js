import Weather from "@/components/Weather";
import {shallowMount} from "@vue/test-utils";

describe('Weather', () => {
    it('renders the component properly',() => {
        const wrapper = shallowMount(Weather)

        expect(wrapper.html()).toMatchSnapshot()
    })
})
