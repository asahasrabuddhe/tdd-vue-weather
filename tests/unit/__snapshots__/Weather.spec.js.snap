// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Weather renders the component properly 1`] = `
<div>
  Weather
</div>
`;
